utils/dev_config/server.dev.config.js

```diff
@@ -23,16 +23,20 @@ module.exports = {
   },
   modules: {
     api: true,
-    client: false,
+    client: true,
```

Comment:
When developing the UI (which this config is intended for), webpack hosts client files. As a result, we disable the module that would host the files here, to avoid confusion (why are my changes not present etc). What is this being enabled for?

Reply:
Artifact of testing, removing now.